Bio/SeqIO/NibIO.py

```diff
@@ -16,17 +16,23 @@
     3 - G
     4 - N (unknown)
 
+As the first bit in a nibble is set if the nucleotide is soft-masked, we
+additionally have
```

Comment:
You need to have a double colon in RST for the next indented block to be a literal block. Or, this would work nicely as a bullet point list:

- ``8`` - t
- ``9`` - c
- ``a`` - a
- ``b`` - g
- ``c`` - n (unknown)

Maybe also try using back-ticks for style?

This also applies to the list above for the non-masked entries, see https://biopython.org/docs/1.74/api/Bio.SeqIO.NibIO.html or https://biopython.org/DIST/docs/api/Bio.SeqIO.NibIO-module.html which are not ideal.

I find an online RST editor useful for this kind of thing, e.g. https://livesphinx.herokuapp.com/